lib/app_store_connect/user_invitation.rb

```diff
@@ -0,0 +1,18 @@
+module AppStoreConnect
+  class UserInvitation
+    def initialize(first_name:, last_name:, email:, roles:)
```

Comment:
Can we make this a `struct` instead of a`class` and name it `UserInvitationCreateRequest` to match the naming of their documentation? 

https://developer.apple.com/documentation/appstoreconnectapi/userinvitationcreaterequest

Reply:
👍 Sure. I think that makes sense.